internal/status/status_test.go

```diff
@@ -0,0 +1,147 @@
+package status
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/lbryio/lbrytv/app/auth"
+	"github.com/lbryio/lbrytv/app/sdkrouter"
+	"github.com/lbryio/lbrytv/app/wallet"
+	"github.com/lbryio/lbrytv/config"
+	"github.com/lbryio/lbrytv/internal/storage"
+	"github.com/lbryio/lbrytv/internal/test"
+	"github.com/lbryio/lbrytv/models"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(m *testing.M) {
+	// These tests requires an environment close to production setup, i.e. no loading from the config
+	config.Override("LbrynetServers", map[string]string{})
+	defer config.RestoreOverridden()
+
+	dbConfig := config.GetDatabase()
+	params := storage.ConnParams{
+		Connection: dbConfig.Connection,
+		DBName:     dbConfig.DBName,
+		Options:    dbConfig.Options,
+	}
+	c, connCleanup := storage.CreateTestConn(params)
+	c.SetDefaultConnection()
+
+	defer connCleanup()
+
+	os.Exit(m.Run())
```

Comment:
this doesn't work. `os.Exit()` will kill the process immediately without running any deferred functions. take a look at your postgres - i bet you have a million temp databases.

look at how i refactored the other tests that use CreateTestConn. you should do it that way